pkg/runner/local_common.go

```diff
@@ -30,28 +29,33 @@ func localCommonHealthcheck(ctx context.Context, hh *healthcheck.Helper, cli *cl
 
 	// prometheus built from Dockerfile.
 	// Check if container exists, if not, build image AND start container.
-	_, exposed, _ := nat.ParsePortSpecs([]string{"9090:9090"})
-	hh.Enlist("local-prometheus",
-		healthcheck.CheckContainerStarted(ctx, ow, cli, "testground-prometheus"),
-		healthcheck.StartContainer(ctx, ow, cli, &docker.EnsureContainerOpts{
-			ContainerName: "testground-prometheus",
-			ContainerConfig: &container.Config{
-				Image: "testground-prometheus:latest",
-			},
-			HostConfig: &container.HostConfig{
-				PortBindings: exposed,
-				NetworkMode:  container.NetworkMode(controlNetworkID),
-			},
-			ImageStrategy: docker.ImageStrategyBuild,
-			BuildImageOpts: &docker.BuildImageOpts{
-				Name:     "testground-prometheus:latest",
-				BuildCtx: filepath.Join(srcdir, "infra/local-docker/testground-prometheus"),
-			},
-		}),
-	)
+	// TODO(raulk): decide what to do with this; the daemon no longer has access
+	//  to testground's source. We are discarding Prometheus for test plan metrics
+	//  anyway; and there are no infrastructure metrics that are valuable to monitor
+	//  in the local runners, so I'm inclined to drop Prometheus entirely from the
+	//  local runners.
+	// _, exposed, _ := nat.ParsePortSpecs([]string{"9090:9090"})
+	// hh.Enlist("local-prometheus",
+	// 	healthcheck.CheckContainerStarted(ctx, ow, cli, "testground-prometheus"),
+	// 	healthcheck.StartContainer(ctx, ow, cli, &docker.EnsureContainerOpts{
+	// 		ContainerName: "testground-prometheus",
+	// 		ContainerConfig: &container.Config{
+	// 			Image: "testground-prometheus:latest",
+	// 		},
+	// 		HostConfig: &container.HostConfig{
+	// 			PortBindings: exposed,
+	// 			NetworkMode:  container.NetworkMode(controlNetworkID),
+	// 		},
+	// 		ImageStrategy: docker.ImageStrategyBuild,
+	// 		BuildImageOpts: &docker.BuildImageOpts{
+	// 			Name:     "testground-prometheus:latest",
+	// 			BuildCtx: filepath.Join(srcdir, "infra/local-docker/testground-prometheus"),
+	// 		},
+	// 	}),
+	// )
```

Comment:
Since we're dropping prometheus for test plan monitoring, I'm inclined to drop it for local executors, as we'd probably use other tools to monitor them.